tests/unit-tests/python/foglamp_test/services/core/api/test_statistics_history.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import asyncpg
+import asyncio
+from datetime import datetime
+import requests
+import pytest
+
+__author__ = "Praveen Garg"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+__DB_NAME = "foglamp"
+BASE_URL = 'http://localhost:8081/foglamp'
+
+pytestmark = pytest.mark.asyncio
+
+
+async def set_statistics_test_data(val):
+    conn = await asyncpg.connect(database=__DB_NAME)
+    await conn.execute('''UPDATE foglamp.statistics SET value = $1, previous_value = $2''', val, 0)
+    await conn.close()
+
+
+@pytest.allure.feature("api")
+@pytest.allure.story("statistics-history")
+class TestStatisticsHistory:
+
+    @classmethod
+    def setup_class(cls):
+        # start foglamp
+        pass
+
+    @classmethod
+    def teardown_class(cls):
+        # stop foglamp
+        pass
+
+    async def test_get_statistics_history(self):
+        r = requests.get(BASE_URL + '/statistics/history')
+        res = r.json()
+
+        assert 200 == r.status_code
+        last_count = len(res['statistics']) * 8
+
+        # use fixture
+        await set_statistics_test_data(10)
+
+        assert 15 == res['interval']
```

Comment:
This may not always  be 15 seconds, it is configurable. We should check it matches the configuration not a fixed value.